packages/desktop/electron/preload.js

```diff
@@ -130,6 +139,7 @@ try {
         },
     })
     contextBridge.exposeInMainWorld('__ELECTRON__', ElectronApi)
+    contextBridge.exposeInMainWorld('__LEDGER__', LedgerApi)
```

Comment:
is ledger the correct term for this? 
I think the apis we currently have ` generateEvmAddress` will be the same for ledger and stronghold accounts?

Reply:
I think it is. We want those API to be available for the FF app and we'll provide some abstraction layer on top of it that handles the different account types.